### PR TITLE
Roll back groups UI release (will need reversion ~Sept 5-9)

### PR DESCRIPTION
**Pull request by zuluecho9**
On Aug 29 the new groups UI was rolled back, so this PR reverts some of the docs changes related to that. When the new groups UI is re-released (estimated week of Sept 5), we can revert this PR and put back those changes.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx [hunk 1]
```diff
@@ -146,21 +146,23 @@ You can manage groups via either the UI or via API:
...
-
-<img title="New Relic user mgmt - create group" alt="New Relic user mgmt - create group" src={createGroupUi}/>
-<figcaption>On the **Groups** tab, you can click **Create new group** to add a user group.</figcaption>
-
-When you're done, if users are already assigned to the group you created or edited, those users will have the new access within a few minutes (although for [EU region organizations](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), this can take up to twenty minutes or so). If your users are not yet in that group (which would be true if you just created a new group), the next step will show you how to add users. 
+1. From the **Access management** UI, click **Access grants and groups**. 
+2. Choose one of the following:
+* **Existing**: If there is already a group you want to add an access grant to, you can use this. For example, if you want to gives users in the default [**Admin** or **User** group](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#groups) access to new accounts, you might choose this and then select the **Admin Default** or **Admin User** role.
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx:153]: @zuluecho9 I would add four spaces to all child-level bullets to get more indentation (or two, two-space tabs). See our [style guide](https://docs.newrelic.com/docs/style-guide/structure/styleguide-markup-indentation#second-and-third) for the nitty gritty.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx [hunk 2]
```diff
@@ -146,21 +146,23 @@ You can manage groups via either the UI or via API:
...
+* **New**: If you need to create a new group, choose this. (This is not available for SCIM provisioning because your groups are managed via SCIM.) 
+3. Next, under **Access grant**, you'll choose one of following:
+* **Account**: Choose this to be able to select from [the roles that are account-scoped](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles). These are the roles that have to do with using and configuring our observability features (and not about organization and user management).
+* **Organization**: Choose this to be able to select from [the roles that are organization-scoped](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles). These are the roles that govern organization- and user management. (Note that these users must also already belong to an account-scoped role. This is true for most users but if it's not, you may see a message that the user doesn't belong to an organization.)
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx:157]: @zuluecho9 do you need the hyphen following "organization"? This doesn't seem like a muti-word adjective.

**Review by rhetoric101 — APPROVED** [review]
@zuluecho9 This looks good! I just inserted a few minor suggestions.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx:153]: oh right. i pasted this from raw version and forgot the indents. thanks

**Pull request merged**